magefiles/magefile.go

```diff
@@ -161,7 +157,7 @@ func getPlugins() error {
 		feed    string
 		version string
 	}{
-		{name: "kubernetes", feed: "https://cdn.porter.sh/plugins/atom.xml", version: "latest"},
```

Comment:
It doesn't look like we are publishing to the latest tag anymore

Reply:
When you run `porter plugin install` and use the latest version, it downloads the atom.xml file and selects the highest version of that plugin, excluding pre-releases.